2024-11-04 12:19:14.660 - INFO    - MainThread - client.py:204 [connectAsync] - Connecting to 127.0.0.1:7497 with clientId 47816...
2024-11-04 12:19:14.662 - INFO    - MainThread - client.py:212 [connectAsync] - Connected
2024-11-04 12:19:14.668 - INFO    - MainThread - client.py:341 [_onSocketHasData] - Logged on to server version 178
2024-11-04 12:19:14.685 - INFO    - MainThread - client.py:218 [connectAsync] - API connection ready
2024-11-04 12:19:14.687 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2104, reqId -1: Market data farm connection is OK:hfarm
2024-11-04 12:19:14.687 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2104, reqId -1: Market data farm connection is OK:usfuture.il
2024-11-04 12:19:14.688 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2104, reqId -1: Market data farm connection is OK:cashfarm
2024-11-04 12:19:14.688 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2104, reqId -1: Market data farm connection is OK:usfuture
2024-11-04 12:19:14.689 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2104, reqId -1: Market data farm connection is OK:usfarm.nj
2024-11-04 12:19:14.689 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2104, reqId -1: Market data farm connection is OK:usfarm
2024-11-04 12:19:14.689 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2106, reqId -1: HMDS data farm connection is OK:euhmds
2024-11-04 12:19:14.690 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2106, reqId -1: HMDS data farm connection is OK:apachmds
2024-11-04 12:19:14.690 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2106, reqId -1: HMDS data farm connection is OK:fundfarm
2024-11-04 12:19:14.691 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2106, reqId -1: HMDS data farm connection is OK:ushmds
2024-11-04 12:19:14.691 - INFO    - MainThread - wrapper.py:1217 [error] - Warning 2158, reqId -1: Sec-def data farm connection is OK:secdefhk
2024-11-04 12:19:15.120 - INFO    - MainThread - ibstore_insync.py:2754 [connectAsync] - Synchronization complete
